contrib/coreos/user-data.example

```diff
@@ -1,25 +1,57 @@
 #cloud-config
 ---
 coreos:
-  etcd:
-    # generate a new token for each unique cluster from https://discovery.etcd.io/new
-    # uncomment the following line and replace it with your discovery URL
-    # discovery: https://discovery.etcd.io/12345693838asdfasfadf13939923
-    addr: $private_ipv4:4001
-    peer-addr: $private_ipv4:7001
-    # give etcd more time if it's under heavy load - prevent leader election thrashing
-    peer-election-timeout: 2000
-    # heartbeat interval should ideally be 1/4 or 1/5 of peer election timeout
-    peer-heartbeat-interval: 500
   fleet:
     # We have to set the public_ip here so this works on Vagrant -- otherwise, Vagrant VMs
     # will all publish the same private IP. This is harmless for cloud providers.
     public-ip: $private_ipv4
     # allow etcd to slow down at times
     etcd_request_timeout: 3.0
+    etcd_servers: http://127.0.0.1:2379
   units:
   - name: etcd.service
+    mask: true
```

Comment:
I was referring to what you said about the `etcd2` section doing nothing until CoreOS >= 653.0.0. The important point is that running this PR leaves you with a cluster that is *not* running `etcd2` in containers or otherwise.